flixel/FlxObject.hx

```diff
@@ -34,48 +35,57 @@ class FlxObject extends FlxBasic
 
 	/**
 	 * Generic value for "left". Used by `facing`, `allowCollisions`, and `touching`.
```

Comment:
I guess you forgot to update `facing`?

Reply:
facing is an interesting case, since it doesn't allow cases like LEFT|RIGHT and ANY, does it seem worthwhile to make a separate enum that uses the same underlying values?

Reply:
in fact, I never use `facing` and the documentation confuses me on whether it expects single directions or values like `LEFT | UP`

particularly this comment:
> Can be set to `LEFT`, `RIGHT`, `UP`, and `DOWN` to take advantage of flipped sprites and/or just track player orientation more easily.

seems to conflict with this in the `setFacingFlip` doc
> E.g. To make a sprite flip horizontally when it is facing both `UP` and `LEFT`, use `setFacingFlip(LEFT | UP, true, false);`

Reply:
Looking at the source, I don't think anything is stopping combined direction flags from working. I could see that being used for sprites with 8 facing directions (including diagonals) rather than just 4. It's probably not very common though.